src/config/bluetooth.md

```diff
@@ -16,70 +16,22 @@ ID TYPE     DEVICE      SOFT      HARD
 
 ## Installation
 
-Install the `bluez` package.
-
-```
-# xbps-install -S bluez
-```
-
-Enable the `bluetoothd` and `dbus` services.
-
-```
-# ln -s /etc/sv/dbus /var/service/
-# ln -s /etc/sv/bluetoothd /var/service/
-```
-
-Add your user to the `bluetooth` group and restart the `dbus` service, or simply
-reboot the system. Note that restarting the `dbus` service may kill processes
-making use of it.
-
-```
-# usermod -a -G bluetooth $USER
-# sv restart dbus
-```
+Install the `bluez` package and enable the `bluetoothd` and `dbus` services.
+Then, add your user to the `bluetooth` group and restart the `dbus` service, or
+simply reboot the system. Note that restarting the `dbus` service may kill
+processes making use of it.
 
 > Note: To use an audio device such as a wireless speaker or headset, ALSA users
 > need to install the `bluez-alsa` package, while
 > [PulseAudio](./media/pulseaudio.md) users do not need any additional software.
 
 ## Usage
 
-Manage Bluetooth connections and controllers using `bluetoothctl`. It uses a
-command line interface; to find out what commands are available, enter `help`.
-To exit, enter `exit` or `quit`.
-
-```
-$ bluetoothctl
-
-[bluetooth]# help
-Menu main:
-Available commands:
--------------------
-advertise             Advertise Options Submenu
-scan                  Scan Options Submenu
-gatt                  Generic Attribute Submenu
-list                  List available controllers
-show [ctrl]           Controller information
-select <ctrl>         Select default controller
-devices               List available devices
-paired-devices        List paired devices
-...
-```
+Manage Bluetooth connections and controllers using `bluetoothctl`, which uses a
+command line interface and also accepts commands from stdin.
```

Comment:
s/uses/provides/? And should 'stdin' be capitalised? (This appears to be the only place it's used in the documentation at the moment.)

Reply:
Fixed both. I capitalised to STDIN, do you think we should make it `STDIN`?

Reply:
Well, i've been looking into how it's written elsewhere, and it looks like lower-case might actually be the way to go: cf. e.g. [this](https://www.gnu.org/software/libc/manual/html_node/Standard-Streams.html) and [this](https://www.gnu.org/software/libc/manual/html_node/Standard-Streams.html). So my Perliness might be showing. :-) i'm slightly inclined towards `stdin` rather than 'stdin', but would be interested to know what others think.

Reply:
I think you pasted the same link twice.

I searched for it a bit, as well. I found STDIN, **STDIN**, stdin and `stdin`. The file descriptors in C are a good reference, and if it comes from code I'd argue for `stdin`.

At least where I could find, the Arch Wiki uses stdin.

Reply:
Gah, sorry! The first link was supposed to be https://pubs.opengroup.org/onlinepubs/009695399/functions/stdin.html.

Reply:
No problem! Hmm... They seem to just use *stdin*, which I wouldn't say is the worst choice available to us.

Reply:
I'd write "also accepts commands on standard input"

Reply:
Yup, changed it! Thanks :)